src/app/core/signup/signup-success/signup-success.component.html

```diff
@@ -0,0 +1,5 @@
+<p>
```

Comment:
Add style for this component after registered show page like login and signup page.